### DISCUSSION TITLE
[keyball61]特定のKEYが同期して動作してしまう。

### DISCUSSION THREAD

**Discussion by hrtpears** [Yowkees/keyball#551] [Q&A]
お世話になります。
テスト用プロファイルで動作確認していますが
基盤の印字番号31(Yキー)と基盤の印字番号19(Oキー)が
同じKEYID(R6C6 KC1c)となってしまいOキーの出力がYキーのものになってしまいます。
原因、解決方法を教えて下さい。

![Okey](https://github.com/Yowkees/keyball/assets/166685050/1df76de3-1336-4652-9fd8-07bdfbbc8674)
![Ykey](https://github.com/Yowkees/keyball/assets/166685050/de407b6c-fec8-4517-b494-25cd2a216c8a)
![kiban](https://github.com/Yowkees/keyball/assets/166685050/d1a2e565-df06-4964-9c01-8eaad16beced)

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

ダイオードの向きの確認と、添付しました写真の赤い丸で示したダイオードのはんだ付けをやり直していただいてもよろしいでしょうか。
![321513140-d1a2e565-df06-4964-9c01-8eaad16beced](https://github.com/Yowkees/keyball/assets/143698349/c34d9c26-369d-4485-8aca-69f4847a73fe)

**Comment by hrtpears** [comment]
赤丸で示して頂いた19のダイオードの向きを凡ミスで逆に付けていたことが原因でした。
予備のダイオードで新しく付け直したところ、正常に動作するようになりました。
的確にサポート頂きまして大変助かりました。ありがとうございます。

**Comment by Ryo624** [comment]
無事に解決していただきましてありがとうございます。
末永くご愛用いただけたら幸いです。
